### PR TITLE
Create project

### PR DESCRIPTION
**Pull request by tuanda5353**

### CODE REVIEW

**Review by DoanVanToan — COMMENTED** [review]
- remove idea files
- add end of file

**Comment by DoanVanToan** [comment]
commented

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: remove hardcode

**Comment by DoanVanToan** [comment]
commented

**Comment by tuanda5353** [comment]
ready

**Comment by tuanda5353** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**